packages/rocketchat-lib/server/functions/notifications/email.js

```diff
@@ -111,14 +102,12 @@ export function sendEmail({ message, user, subscription, room, emailAddress, has
 
 	const link = getMessageLink(room, subscription);
 
-	if (RocketChat.settings.get('Direct_Reply_Enable')) {
-		contentFooter = RocketChat.placeholders.replace(RocketChat.settings.get('Email_Footer_Direct_Reply') || '');
-	}
+	const directReply = RocketChat.settings.get('Direct_Reply_Enable') ? '' : ''; // TODO templatemail
```

Comment:
Why is there a `TODO` here?
Do you need a migration to `Email_Footer_Direct_Reply` lose their footer-related content?